components/calendar/src/astronomy.rs

```diff
@@ -1542,7 +1540,7 @@ impl Astronomical {
     ///
```

Comment:
nit: docs should mention `julian_centuries`

Reply:
done.